v2/samples/authorization/v1beta/v1beta20200801preview_roleassignment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: authorization.azure.com/v1beta20200801preview
 kind: RoleAssignment
 metadata:
-  name: b5ce4677-d53d-4532-aa18-ac32aecb1117 # Should be UUID
+  name: fee8b6b1-fe6e-481d-8330-0e950e4e6b86 # Should be UUID
```

Comment:
Once my other PR merges I can update these samples to use a dynamically provisioned identity and avoid hardcoding principalId